CONTRIBUTING.md

```diff
@@ -173,15 +173,15 @@ You will make changes to copies of thefiles which make up rambox in a personal f
 
 #### Important: ALWAYS EDIT ON A BRANCH
 
-Take away only one thing from this document: Never, **EVER** make edits to the `staging` branch.
+Take away only one thing from this document: Never, **EVER** make edits to the `master` branch.
 ALWAYS make a new branch BEFORE you edit files.
-This is critical, because if your PR is not accepted, your copy of staging will be forever sullied and the only way to fix it is to delete your fork and re-fork.
+This is critical, because your copy of `master` will be forever sullied and the only way to fix it is a difficult hard-reset and force-push process, or deleting your fork and re-forking.
 
 ### Common Steps
 
 1. Once the edits have been committed, you will be prompted to create a pull request on your fork's GitHub Page.
-1. By default, all pull requests should be against the rambox main repo, `staging` branch.
-    - **Make sure that your Base Fork is set to saenzramiro/rambox when raising a Pull Request.**
+1. By default, all pull requests should be against the rambox main repo, `master` branch.
```

Comment:
It looks like you use `master` instead of `staging` now - I couldn't find a `staging` branch, and it looks like most activity in this repo is in `master`.